deeppavlov/models/doc_retrieval/logit_ranker.py

```diff
@@ -33,50 +32,69 @@ class LogitRanker(Component):
         squad_model: a loaded squad model
         batch_size: batch size to use with squad model
         sort_noans: whether to downgrade noans tokens in the most possible answers
+        top_n: number of answers to return
 
      Attributes:
         squad_model: a loaded squad model
         batch_size: batch size to use with squad model
+        top_n: number of answers to return
 
     """
 
     def __init__(self, squad_model: Union[Chainer, Component], batch_size: int = 50,
-                 sort_noans: bool = False, **kwargs):
+                 sort_noans: bool = False, top_n: int = 1, **kwargs):
         self.squad_model = squad_model
         self.batch_size = batch_size
         self.sort_noans = sort_noans
+        self.top_n = top_n
+
+    def __call__(self, contexts_batch: List[List[str]], questions_batch: List[List[str]],
+                 tfidf_doc_ids: Optional[List[List[str]]] = None) -> \
+            Union[Tuple[List[str], List[float], List[int], List[str]],
+                  Tuple[List[List[str]], List[List[float]], List[List[int]], List[List[str]]]]:
 
-    def __call__(self, contexts_batch: List[List[str]], questions_batch: List[List[str]]) -> \
-            Tuple[List[str], List[float]]:
         """
         Sort obtained results from squad reader by logits and get the answer with a maximum logit.
 
         Args:
             contexts_batch: a batch of contexts which should be treated as a single batch in the outer JSON config
             questions_batch: a batch of questions which should be treated as a single batch in the outer JSON config
-
+            tfidf_doc_ids (optional): names of the documents from which the contexts_batch was derived
         Returns:
-            a batch of best answers and their scores
-
+             a batch of best answers, their scores, places in contexts
+             and tfidf_doc_ids for this answers if tfidf_doc_ids were passed
         """
-        # TODO output result for top_n
-        warnings.warn(f'{self.__class__.__name__}.__call__() API will be changed in the future release.'
-                      ' Instead of returning Tuple(List[str], List[float] will return'
-                      ' Tuple(List[List[str]], List[List[float]]).', FutureWarning)
+        if tfidf_doc_ids is None:
+            logger.info("you didn't pass tfidf_doc_ids as input in logit_ranker config so batch_best_answers_doc_ids"
+                        "can't be compute")
```

Comment:
info -> warning
and add space after batch_best_answers_doc_ids
```suggestion
            logger.warning("you didn't pass tfidf_doc_ids as input in logit_ranker config so "
                           "batch_best_answers_doc_ids can't be compute")
```